lib/interpreter/instantiate/function.cpp

```diff
@@ -23,6 +23,11 @@ Expect<void> Interpreter::instantiate(
     auto NewFuncInst = std::make_unique<Runtime::Instance::FunctionInstance>(
         ModInst.Addr, *FuncType, CodeSegs[I]->getLocals(),
         CodeSegs[I]->getInstrs());
+    NewFuncInst->setSymbol(CodeSegs[I]->getSymbol());
```

Comment:
I have no idea about this line and the following lines.
It looks like 26 & (28-30) are doing the same stuff but the above one doesn't check the Symbol before calling the `setSymbol` function

Reply:
Looks like I forgot to remove this line.